docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -25,6 +25,15 @@ configured in group policy.
 [Read more about exporting the Teleport CA](../../desktop-access/manual-setup.mdx#export-the-teleport-ca)
 </Admonition>
 
+<Admonition type="warning" title="Certificate Authority Types">
+Prior to Teleport v12 if `--type` was unspecified it would default to rotating
+all certificate authorities. This is known to cause issues and has been deprecated.
```

Comment:
I wouldn't say _known to cause issues_. How about something like this instead?

Prior to Teleport 12, `--type` would default to rotating all certificate authorities.
It is best to rotate CAs one at a time for increased stability.
Future versions of Teleport will require this, but you can opt in to the previous behavior by adding an explicit `--type=all` flag.

Reply:
Done.